src/Microsoft.DotNet.Cli.Utils/project.json

```diff
@@ -2,7 +2,13 @@
   "version": "1.0.0-preview3-*",
   "buildOptions": {
     "keyFile": "../../tools/Key.snk",
-    "warningsAsErrors": true
+    "warningsAsErrors": true,
+    "compile": {
+      "include": [
+        "**/*.cs",
+        "../../src/dotnet/DotnetFiles.cs"
```

Comment:
I am not sure how to work around it. Without this, the tests won't work on Sierra. This is an internal file, therefore it is not visible outside to consumer of the package, right?